tour-of-go/exports/booking/booking_parser.go

```diff
@@ -2,9 +2,9 @@ package booking
 
 import "fmt"
 
-var booking = "This ticket have been booked"
+var booking string = "This ticket have been booked"
 
-func bookedTicket() {
+func getbookedTicket() {
```

Comment:
Correct the this. change it to `getBookedTicket()"

All new words after the first word in a function name must start with capital letter.

Reply:
Done